prody/atomic/atomgroup.py

```diff
@@ -493,7 +525,7 @@ def _getCoords(self):
         if self._coords is not None:
             return self._coords[self._acsi]
 
-    def setCoords(self, coords, label=''):
+    def setCoords(self, coords, label='', overwrite=False):
```

Comment:
it could help to add docs for this

Reply:
Here I only exposed the overwrite=False optional argument present in the _setCoords() signature. _setCoords() is called by setCoords() but I could not pass overwrite=True. In _setCoords, overwrite can be used to force reshaping the _coords array, although the argument is not mentioned i the _setCoords() method doc string either. An alternative would have been for me to set ag1._coords to None prior to calling setCoords() but I felt exposing the argument was cleaner.

Reply:
Yes, but seeing as most users probably aren’t using _setCoords much it makes sense to add this to setCoords docs too

Reply:
I added the  description in the doc string